settings/fontFamilies.js

```diff
@@ -0,0 +1,3 @@
+export const BASE = '\'Open Sans\', \'Helvetica Neue\', \'Helvetica\', \'Arial\', sans-serif'
```

Comment:
If you use template strings instead, you don't have to bother with escaping those quotes.